### PR TITLE
release: bump version to 1.8.0

### PR DESCRIPTION
**Pull request by akshetpandey**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,14 +1,12 @@
 [tool.poetry]
 name = "poetry-plugin-export"
-version = "1.6.0"
+version = "1.8.0"
```
Comment (dimbleby) [inline pyproject.toml:3]: why not 1.7.0?  there is no need to keep the plugin version synchronized with the poetry version

skipping a version will just cause people to raise issues asking why

File: pyproject.toml [hunk 2]
```diff
@@ -20,8 +18,7 @@ include = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
-poetry = "^1.6.0"
-poetry-core = "^1.7.0"
+poetry = "^1.8.0"
```
Comment (dimbleby) [inline pyproject.toml:21]: this plugin imports directly from poetry-core, so it should explicitly depend on it

Comment (akshetpandey) [reply pyproject.toml:21]: Ahh, I was trying to find the import but didn't see it, let me fix that.

Comment (akshetpandey) [reply pyproject.toml:21]: Fixed

File: pyproject.toml (hunk 1 above)
Comment (akshetpandey) [reply pyproject.toml]: Changed it to 1.7.0

**Pull request closed without merging**